app/renderer/js/components/webview.ts

```diff
@@ -147,7 +148,16 @@ class WebView extends BaseComponent {
 		});
 	}
 
+	updateBadgeCount(count: number): void {
+		this.badgeCount = count;
+		this.props.onTitleChange();
```

Comment:
Would it be a good idea to keep updating `domain.json` with the correct badge count here too so that when the user switches on notifications again, they get the correct number immediately even if the page hasn't loaded?

Reply:
the `updateBadgeCount()` in `webview.ts` is used by the muting the feature, so the value at the `domain.json` will have no effect on adding it to the function. Further, the function itself is dependent on the called parameters, so there will be no changes.

Reply:
THis might not be a very good idea. THe badgecount in the title changes when we scroll down a page and accessing the json-db this many times, is not a good idea.

Reply:
The json-db will be accessed only at the times of muting and unmuting an organization. Once its unmuted, it will update the value according to the present number of unread messages.